### PR TITLE
AWS circleci support

### PR DESCRIPTION
**Pull request by artem-belov**
CircleCI integration-testing job using AWS EKS cluster 

Signed-off-by: Artem Belov <artem.belov@xored.com>

## Description
AWS commands added to CircleCI jobs

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by artem-belov** [comment]
#1002 need to be merged first to pass all AWS tests

**Comment by edwarnicke** [comment]
@artem-belov This appears to need a manual rebase onto master.  I've disabled Azure for now until I can resolve the issues with quotas there.   So the Azure testing once rebased won't block you.

**Comment by edwarnicke** [comment]
@artem-belov Still seems not rebased on master so Azure is still failing

**Comment by haiodo** [comment]
@artem-belov Could you please re-trigger build, since packet net was issues today, so tests are not executed for this PR.

File: scripts/aws/create-kubernetes-cluster.go [hunk 1]
```diff
@@ -236,8 +237,8 @@ func createEksWorkerNodes(cfClient *cloudformation.CloudFormation, nodesStackNam
 				ParameterValue: keyPairName,
 			},
 			{
-				ParameterKey: aws.String("NodeImageId"),
-				ParameterValue: aws.String("ami-0484545fe7d3da96f"),
+				ParameterKey:   aws.String("NodeImageId"),
+				ParameterValue: aws.String("ami-04ea7cb66af82ae4a"),
```
Comment (haiodo) [inline scripts/aws/create-kubernetes-cluster.go]: @artem-belov could you please clarify of this constants. Thanks.

**Comment by artem-belov** [comment]
Depend on #1039

File: scripts/aws/create-kubernetes-cluster.go [hunk 2]
```diff
@@ -226,6 +227,11 @@ func createEksWorkerNodes(cfClient *cloudformation.CloudFormation, nodesStackNam
 
 	s := string(sf)
 
+	// Base image for Amazon EKS worker nodes
+	// with Kubernetes version 1.12.7
+	// for region us-east-2
+	eksAmi := aws.String("ami-04ea7cb66af82ae4a")
```
Comment (haiodo) [inline scripts/aws/create-kubernetes-cluster.go]: Should we move this to some kind of config?

Comment (haiodo) [reply scripts/aws/create-kubernetes-cluster.go]: Should we comment at least there to get it?

https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html

**Comment by haiodo** [comment]
@nickolaev @edwarnicke We waiting to increase number of available clusters per region in AWS before  we will be merging this PR.

**Comment by edwarnicke** [comment]
@haiodo I increased AWS VPCs in the us-east-2 region (where we are running).  Since master has settled down, I’m going to merge this :)

**Pull request merged**